imisi-akande/postit_site
# User should be able to search for posts

What does this PR do?

This PR add feature for a user to be able to search for post(s)

Description of Task to be completed?

created a filter method for user to be able to filter through post(s)


How should this be manually tested?

by
clone the repo
run python manage.py runserver
the app starts at local-host 8000 by default
to search for a post title of "Technology" - type http://127.0.0.1:8000/api/posts/?q=Technology

Any background context you want to provide?

N/A

What are the relevant pivotal tracker stories?

a user should be able to search for post(s)

Screenshots (if appropriate)

N/A

Questions:

N/A
